clustering/ambari_cluster_state.py

```diff
@@ -0,0 +1,387 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Author: Mark Bittmann (https://github.com/mbittmann)
+# Documentation section
+DOCUMENTATION = '''
+---
+module: ambari_cluster_state
+version_added: "2.0"
+author: Mark Bittmann (https://github.com/mbittmann)
+short_description: Create, delete, start or stop an ambari cluster
+description:  
+    - Create, delete, start or stop an ambari cluster
+options:
+  host:
+    description:
+      The hostname for the ambari web server
+  port:
+    description:
+      The port for the ambari web server
+  username:
+    description:
+      The username for the ambari web server
+  password:
+    description:
+      The name of the cluster in web server
+    required: yes
+  cluster_name:
+    description:
+      The name of the cluster in ambari
+    required: yes
+  cluster_state:
+    description:
+      The desired state for the ambari cluster ['present', 'absent', 'started', 'stopped']. Setting the cluster
+      state to absent will first stop the cluster.
+    required: yes
+  blueprint_yml:
+    description:
+      The path to the file defining the cluster blueprint and host mapping. Required when state == 'present'
+    required: no
+  blueprint_name:
+    description:
+      The name of the blueprint. Required when state == 'present'
+    required: no
+  wait_for_complete:
+    description:
+      Whether to wait for the request to complete before returning. Default is False.
+    required: no
```

Comment:
Add `requirements`: yaml and requests ((See)[https://github.com/ansible/ansible-modules-extras/blob/devel/cloud/cloudstack/cs_facts.py#L44])